src/pages/create-todo-page.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+import { CreateTodoContent } from 'components/create-todo-content';
+import * as S from 'styles/styled';
+
+export function CreateToDoPage() {
+  return <S.Container>{<CreateTodoContent />}</S.Container>;
```

Comment:
esse `{<CreateTodoContent />}` é desnecessário, usamos chaves apenas pra renderizar valores que não são elementos html ou jsx, nesse caso poderia ser apenas isso:
```suggestion
  return <S.Container><CreateTodoContent /></S.Container>;
```